### ISSUE TITLE
AcademicYear needs some rethinking

### ISSUE THREAD

**Issue by bgotink** [LitusProject/Litus#215]
AcademicYear ~~sucks~~ needs some rethinking.

Right now, there are a buttload of (possible) issues due to the dual nature of the AcademicYear entity. It represents both the academic year and the organization year. This gives some funny results.

Exhibit A:
@dwendelen can still access the 6, even though he is no longer praesidium.
Why? Because the [ApiController](https://github.com/LitusProject/Litus/blob/master/module/ApiBundle/Component/Controller/ActionController/ApiController.php#L358-L369) superclass uses the academic year, while the admin uses the organization year.
Daan's praesidium-status will be revoked for the door on 22 September, rather than 15 July.

In the mean time, the new people cannot enter, because their praesidium status hasn't gone into effect yet. [rumours, can you configm this, @koencertyn?]

Exhibit B:
It's 8 September. This falls _before_ the end of organization year 2013-2014, because that only ends when the academic year ends [[proof](https://github.com/LitusProject/Litus/blob/master/module/CommonBundle/Entity/General/AcademicYear.php#L87-L90)]. It also falls _after_ the start of organization year 2014-2015. Which is it?
~~Even if the bug in exhibit A is fixed (by using the organization year instead of the academic year), this still means @dwendelen can enter the 6 at will, because he's still praesidium.~~ This bug just never gets noticed because the `CommonBundle\Component\Util\AcademicYear` utility uses the right end date.

**TL;DR**
We need to rethink this. There's _no_ legitimate reason why admins should be able to make someone student in an academic year that hasn't even started yet, and there's _no_ reason why new praesidium members should still be shown as non-praesidium in their profile.

Is there a clean way to separate the two? Or can we unify the two in a clean way? Whatever this current state is called, it's messy.
